src/main/resources/application.properties

```diff
@@ -1 +1,4 @@
 spring.application.name=GitHubAiAPI
+
+
```

Comment:
It's a good practice to use meaningful names for your properties. 'Spring.demo.value' doesn't provide much context about its purpose. Consider renaming it to something more descriptive.